### PR TITLE
Update the W/X quickstart articles + remove preview labels of previous articles when the Dev Kit GA

### PR DESCRIPTION
**Pull request by MingjiaLiu1995**
1. Add the Dev Kit option in the Excel quickstart article.
2. Add the Dev Kit option in the Word quickstart article.
3. Remove the preview labels in previous articles. (New updates)

The articles when NOT be published until the Dev Kit is ready for GA.

The pr is a follow-up of the previous pr #4902 to preview the document change.

### CODE REVIEW

**Comment by MingjiaLiu1995** [comment]
@Rick-Kirkham I changed source version to the new branch. Is there a way I can preview the document from here? Thanks!

**Comment by Rick-Kirkham** [comment]
> @Rick-Kirkham I changed source version to the new branch. Is there a way I can preview the document from here? Thanks!

Yes. The way to see the preview is to navigate to the existing page. Then add the word "review" to the beginning of the domain in the address bar. When the page on review.learn.microsoft.com opens, there will be a dropdown menu just above the tiitle of the article where you can select your branch. In this case "DevKitGAUpdate". This will open a preview version with the changes in that branch:

https://review.learn.microsoft.com/en-us/office/dev/add-ins/develop/development-kit-overview?branch=DevKitGAUpdate

Then in that same browser tab you can navigate to any other pages you want and you'll see the version of them in your branch.

File: docs/develop/development-kit-overview.md [hunk 1]
```diff
@@ -1,21 +1,17 @@
 ---
-title: Create Office Add-in projects using Office Add-ins Development Kit for Visual Studio Code (preview)
+title: Create Office Add-in projects using Office Add-ins Development Kit for Visual Studio Code 
 description: Learn how to create Office Add-in projects using Office Add-ins Development Kit extension.
-ms.date: 09/09/2024
+ms.date: 11/12/2024
```
Comment (Rick-Kirkham) [inline docs/develop/development-kit-overview.md:4]: the ms.date value has to be American style, with the month first and then the day of the month. Also, be sure to update the ms.date on the two quickstart articles and the article about setting up the developer environment.

```suggestion
ms.date: 12/11/2024
```

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

File: docs/develop/development-kit-overview.md (hunk 1 above)
Comment (MingjiaLiu1995) [reply docs/develop/development-kit-overview.md]: Done

**Review by Rick-Kirkham — APPROVED** [review]

**Pull request merged**